ursina/shaders/lit_with_shadows_shader.py

```diff
@@ -1,4 +1,6 @@
-from ursina import *; lit_with_shadows_shader = Shader(language=Shader.GLSL, name='lit_with_shadows_shader', vertex = '''#version 150
+from ursina import *
+
+lit_with_shadows_shader = Shader(language=Shader.GLSL, name='lit_with_shadows_shader', vertex = '''#version 150
```

Comment:
The reason this was all on a single line was so that when you get an error, the line number from the error will match the file's line numbers. In this case you'd have to subtract by 2 to find it.